src/devtools/server/actors/highlighters/box-model.js

```diff
@@ -522,84 +522,85 @@ class BoxModelHighlighter extends AutoRefreshHighlighter {
 
   _getBoxPathCoordinates(boxQuad, nextBoxQuad) {
     const { p1, p2, p3, p4 } = boxQuad;
+    const scale = getDevicePixelRatio();
```

Comment:
Might be worth adding a `scaleQuad` method that does the calculations and returns a new quad. Would reduce the diff here and be easier to follow I think.